selenium/pom.xml

```diff
@@ -118,7 +118,7 @@
         <browserstack.automateKey>hp3vg22JUprFFZzESJrz</browserstack.automateKey>
         <browserstack.username>developers44</browserstack.username>
         <browserstack.local>false</browserstack.local>
-        <aodnPortal>https://portal-systest.aodn.org.au/</aodnPortal>
+        <aodnPortal>https://portal-edge.aodn.org.au/</aodnPortal>
```

Comment:
OK so I also notice that we are publishing our username and automateKey details in a public repo  - is this an issue?